src/App.tsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import {Weather} from './components/weather/Weather';
-import s from './App.module.css';
+import style from './App.module.css';
+import {News} from './components/news/News';
+
+enum WidgetTitles {
+    WeatherWidget = 'weather-widget',
+    NewsWidget = 'news-widget',
+}
 
 type PropsType = {
-    app?: string;
+    app?: WidgetTitles;
     city?: string;
+    country?: string;
 };
 
-export const App: React.FC<PropsType> = (props) => {
-
-    enum WidgetTitles {
-        WeatherWidget = 'weather-widget',
-    };
-
-    return <div className={s.wrapper}>
-        {props.app === WidgetTitles.WeatherWidget &&
-        <Weather city={props.city} />}
-    </div>;
-};
+export const App: React.FC<PropsType> = (props) => (
```

Comment:
App should return necessary component only (according to the props.app). Leave switch() only